Story/StoryHooks.cs

```diff
@@ -249,6 +272,10 @@ private void TextPrompt_Update(On.HUD.TextPrompt.orig_Update orig, TextPrompt se
                     if (touchedInput)
                     {
                         self.gameOverMode = false;
+                        if (OnlineManager.lobby.isOwner)
+                        {
+                            story.wantsToRestartCycle = true;
```

Comment:
also this variable name is misleading, imo `showRestartButton` would be a better name